specification/web/resource-manager/Microsoft.Web/stable/2024-11-01/ResourceProvider.json

```diff
@@ -545,6 +545,98 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/listVirtualNetworkIntegrations": {
+      "post": {
+        "summary": "Retrieve information regarding the provided virtual network integration",
+        "description": "Retrieve information regarding the provided virtual network integration, such as its allocated resources and its allocated IP",
+        "operationId": "VirtualNetworkIntegrations",
+        "parameters": [
+          {
+            "name": "request",
+            "in": "body",
+            "description": "Object contains the subnetResourceId",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VirtualNetworkIntegrationRequest"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
```

Comment:
>apiVersionParameter [](http://example.com/codeflow?start=34&length=19)

you should use the common-types from [https://github.com/Azure/azure-rest-api-specs/blob/680f81b6ef2a6f0216aaa77e3ef5a2cb680b00bb/specification/common-types/resource-management/v5/types.json](https://github.com/Azure/azure-rest-api-specs/blob/680f81b6ef2a6f0216aaa77e3ef5a2cb680b00bb/specification/common-types/resource-management/v5/types.json) as well